lib/internal/freeze_intrinsics.js

```diff
@@ -0,0 +1,232 @@
+// Adapted from SES/Caja - Copyright (C) 2011 Google Inc.
+// Copyright (C) 2018 Agoric
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// based upon:
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/startSES.js
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/repairES5.js
+// then copied from proposal-frozen-realms deep-freeze.js
+'use strict';
+const assert = require('assert');
+
+const intrinsics = [
+  // Anonymous Intrinsics
+  // ThrowTypeError
+  // ?
+  // IteratorPrototype
+  // ?
+  // ArrayIteratorPrototype
+  (new Array())[Symbol.iterator]().__proto__, // eslint-disable-line
+  // StringIteratorPrototype
+  (new String())[Symbol.iterator]().__proto__, // eslint-disable-line
+  // MapIteratorPrototype
+  (new Map())[Symbol.iterator]().__proto__, // eslint-disable-line
+  // SetIteratorPrototype
+  (new Set())[Symbol.iterator]().__proto__, // eslint-disable-line
+  // GeneratorFunction
+  function * x() {}.prototype,
+  // TypedArray
```

Comment:
Stray comment? Typed arrays are listed starting at line 80.

Reply:
This is an anonymous intrinsic actually, reference fixed up thanks to @devsnek help.